scheduler/core/builder/schedulerbuilder.py

```diff
@@ -48,13 +48,11 @@ def build_collector(self,
     def build_selector(collector: Collector,
                        num_nights_to_schedule: int,
                        blueprint: SelectorBlueprint,
-                       cc_per_site: Optional[Dict[Site, CloudCover]] = None,
-                       iq_per_site: Optional[Dict[Site, ImageQuality]] = None) -> Selector:
+                       variant_per_site: Optional[Dict[Site, Variant]] = None) -> Selector:
```

Comment:
This should be `VariantSnapshot` and not `Variant`. `Variant` represents weather conditions across a number of timeslots, whereas `VariantSnapshot` just represents weather conditions, which is what you want, I think.